modules/monitoring/application/controllers/TacticalController.php

```diff
@@ -103,4 +114,15 @@ public function indexAction()
             ->render();
         $this->view->statusSummary = $summary;
     }
+
+
+    /**
+     * Return all tabs for this controller
+     *
+     * @return Tabs
+     */
+    private function createTabs()
```

Comment:
Please use `protected` instead. `private` is only for very specific use-cases. This isn't one of these.

Reply:
The same argument as above; it is private in the ListController...
Should I change both or would you rather leave it untouched there?

Reply:
The same answer as above. The ListController is old. ;)

And no, just change your own code. The ListController is fine for now, even if ancient.